### DISCUSSION TITLE
How can I check which version of Next.js I am working with?

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#39726] [Help] [answered]
How can I check it?  and how can I update it? 

I was trying to use next-pwa and just didnt work.... one thing in their documents said I needed to work with Next 9 or higher?... 

Either way... I am looking to do a PWA without having to install a plugin for it, but would be nice to keep Next updated...

I do have a few plugins installed, but I am trying to keep it to a minimum so I don't run into updating problems later with things that are not compatible.

**Comment by gaston-flores** [comment, marked as answer]
You can check that in your `package.json`, it will say something like `"next": "latest"` or `"next": "^12.2.5"`. You can see the actual version it resolved to in your `package-lock.json` or `yarn.lock`.

**Reply by Elindo586** [reply]
I have "next": "12.1.6",  in package json

Hmmm...

Is there a good way to update it?   and... would it screw up my app ?.....   like I said... I have a few plugins...

**Reply by gaston-flores** [reply]
It's unlikely to break anything for such a small version jump. Do `yarn add next` or `npm install next` and it will install the latest stable version.

**Reply by Elindo586** [reply]
Thanks.... didn't break anything...  lucky me...
